db_cost_aws-bs.md

```diff
@@ -282,13 +289,14 @@ Portworx Perspective consists of the following assets/resources in AWS
 2. 9 EBS volumes 
 * 3 EBS Volumes (non-encrypted) with 20GBs each attached to a corresponding EC2 instance. Each volume is gp2 with 100 IOPS plus
 * 3 redundant volumes in a cluster to support etcd for the cluster @ 150GB per volume
-* 3 EBS volumes connected to the 3 EKS EC2 connected volumes at 20GB each
+* 3 EBS volumes connected to the 3 EKS EC2 connected volumes at 200GB each for Portworx Storage Pool
 3. 2 ELBs One for the application and one for postgres 
 
 RDS Perspective consists of the following assets/resources in AWS
 
-* 4 EC2 instances - 3 t3.large instances for the EKS cluster and one for RDS instance
+* 6 EC2 instances - 3 t3.large instances for the EKS cluster and 3 for Multi-Zone RDS instance
```

Comment:
There are 6 EC2 nodes. 3 for EKS and 3 for multi-az RDS, so i think this line was incorrect.